frontend/src/pages/auth/signup/styles.css

```diff
@@ -0,0 +1,46 @@
+.box {
```

Comment:
1) find better name, for other styles as well.
2) some of the styles are going to be reused in SingIn component. Move it to auth folder